docs/site/Model.md

```diff
@@ -136,16 +136,16 @@ forbid additional properties that are not specified in the type definition. The
 persistence layer respects this constraint and configures underlying
 PersistedModel classes to enforce `strict` mode.
 
-To create a model that allows both well-defined but also arbitrary extra
-properties, you need to disable `strict` mode in model settings through the CLI
+LB4 supports creating a model that allows both well-defined but also arbitrary extra
+properties for **NoSQL** databases such as MongoDB. You need to disable `strict` mode in model settings through the CLI
```

Comment:
I know this isn't part of your changes, but since the change can't *only* be done through the CLI, can you mention you can also do it through adding `{strict: false}` to the model settings manually or say that it *can be* done through the CLI?

Reply:
Good point